mutiny-core/src/socket.rs

```diff
@@ -404,7 +403,9 @@ pub(crate) fn schedule_descriptor_read(
                             error!("got connection error");
                         }
                         gloo_net::websocket::WebSocketError::ConnectionClose(e) => match e.code {
-                            1000 => debug!("normal connection closure"),
+                            // TODO make connection closes from proxy be 1000
+                            1000 => trace!("normal connection closure"),
+                            1006 => debug!("abnormal connection closure"),
```

Comment:
Would be good to make these an enum in the ln-websocket-proxy library eventually

Reply:
It's a underlying WS library thing, and that lib doesn't make them `enum`, have to look up the code online.